view/statistics-flow-certificates.js

```diff
@@ -133,6 +138,11 @@ exports['statistics-main'] = function () {
 					name: 'dateTo', value: location.query.get('dateTo') })
 			),
 			selectPeriodMode(),
+			div(
+				a({ class: 'users-table-filter-bar-print', href:
+					getDynamicUrl('/flow-certificates-data.pdf', { only: params }),
```

Comment:
We should not propagate `page` in query, and PDF should display all records from given period (not only the page that is presented in HTML view). Is that ensured?